### ISSUE TITLE
incorrect response from cloneElement

### ISSUE THREAD

**Issue by maxguzenski** [developit/preact#310]
```javascript
React.cloneElement(<div/>, {}, [<div>1</div>, <div>2</div>])
```

Preact is returning **\<div\>[object Object],[object Object]\</div\>** from above code...

**Comment by developit** [comment]
@maxguzenski Thanks for the report!  I think this is a duplicate of #279.  

```
// produces correct result:
React.cloneElement(<div/>, {}, <div>1</div>, <div>2</div>)

// doubly-nested children fails:
React.cloneElement(<div/>, {}, [<div>1</div>, <div>2</div>])
```

I'll keep this open to track, but bear in mind it's not specific to `cloneElement()`, but applies to all children.

**Comment by developit** [comment]
Fixed in 6.1!
